src/directives/google-map.ts

```diff
@@ -191,10 +191,12 @@ export class SebmGoogleMap implements OnChanges,
     });
   }
 
+  get wrapper(): GoogleMapsAPIWrapper { return this._mapsWrapper; }
```

Comment:
@SebastianM Yes, but it's not the same instance of the GoogleMapsAPIWrapper as the one used by the map.  If you call getMap or anything else on it it fails.  This is because the provider is specified in the map component....